crates/infra/thoughts-core/README.md

```
@@ -237,13 +223,13 @@ The tool organizes all mounts into three distinct spaces:
 ### Platform Abstraction
 The tool automatically detects your platform and uses the appropriate mount technology:
 - **Linux**: Uses mergerfs for high-performance union filesystem
-- **macOS**: Uses fuse-t for FUSE support on Apple Silicon and Intel Macs
+- **macOS**: Uses FUSE-T or macFUSE (prefers FUSE-T when both are present), plus `unionfs-fuse` for the union layer
 
 ### Mount Resolution
```

markdownlint-cli2 (0.22.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)